src/interfaces/pyo3/py_cover_crypt.rs

```diff
@@ -0,0 +1,404 @@
+use abe_policy::AccessPolicy;
+use cosmian_crypto_core::{
+    bytes_ser_de::{Deserializer, Serializable, Serializer},
+    KeyTrait,
+};
+use pyo3::{exceptions::PyException, exceptions::PyTypeError, prelude::*, types::PyType};
+
+use crate::{
+    api::CoverCrypt as CoverCryptRust,
+    interfaces::pyo3::py_abe_policy::Policy,
+    interfaces::statics::{
+        CoverCryptX25519Aes256, EncryptedHeader, MasterSecretKey as MasterSecretKeyRust,
+        PublicKey as PublicKeyRust, SymmetricKey as SymmetricKeyRust,
+        UserSecretKey as UserSecretKeyRust,
+    },
+};
+
+// Pyo3 doc on classes
+// https://pyo3.rs/v0.16.2/class.html
+
+#[pyclass]
+pub struct MasterSecretKey {
+    inner: MasterSecretKeyRust,
+}
+
+#[pymethods]
+impl MasterSecretKey {
+    /// Convert to bytes for sending/saving the key
```

Comment:
The user will define his needs by himself, no need to mention why one would want to convert to bytes here :)